packages/expo-constants/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- If `manifest` is undefined, try to grab it from `expo-updates` ([#10668](https://github.com/expo/expo/pull/10668) by [@esamelson](https://github.com/esamelson))
```

Comment:
```suggestion
- If `manifest` is defined on `expo-updates` then use it instead of `ExponentConstants.manifest` ([#10668](https://github.com/expo/expo/pull/10668) by [@esamelson](https://github.com/esamelson))
```